devices/Technicolor

```diff
@@ -0,0 +1,43 @@
+const fz = require('zigbee-herdsman-converters/converters/fromZigbee');
+const tz = require('zigbee-herdsman-converters/converters/toZigbee');
+const exposes = require('zigbee-herdsman-converters/lib/exposes');
+const reporting = require('zigbee-herdsman-converters/lib/reporting');
+const extend = require('zigbee-herdsman-converters/lib/extend');
+const e = exposes.presets;
+const ea = exposes.access;
+
+module.exports = [
+    {
+        zigbeeModel: ['TKA105'],
+        model: 'XHK1-TC',
+        vendor: 'Technicolor',
```

Comment:
This looks to be an xfinity product (https://www.xfinity.com/support/articles/installing-batteries-xhk1-ue-keypad), can you move it to the correct file and change the vendor here?

Reply:
Shure if you want so.

At the first connection the pad sent technicolor as manufacturer in zigbee2mqtt. If it is no problem that the manufacturer in the .js files is different compared to what the message to zigbee2mqtt says please let me know and I'll change it.

Reply:
Yes, please change to xfinity, it's preferred to have something recognizable.